### ISSUE TITLE
Jest Builder: Unable to use collectCoverageFrom

### ISSUE THREAD

**Issue by emiljohnsen** [meltedspark/angular-builders#198]
**Describe the bug**

Hello! I'm not sure if this is the right place, but I'm not really sure where the problem arises from either. 

**Background:** 
I just set up a new Angular project, removed Karma, and installed Jest, using the Jest builder using a few guides. I provide configuration to Jest via `angular.json`, as advised by the [README](https://github.com/meltedspark/angular-builders/blob/master/packages/jest/README.md). 

I have tried to provide config via `jest.config.js` as well as via package.json, to no avail. I have cleared the cache between runs, and ran it with `--no-cache`. 

**Issue:**
1. Setting `collectCoverageFrom` does not seem to affect Jest. Setting it to an invalid value gives a schema error. 
2. Running `ng test -- --collectCoverageFrom='[/* glob here */]'` in various forms returns `Unknown option: '--collectCoverageFrom'`
3. Running `jest --collectCoverageFrom='[/* glob here */]'` effectively sets the option, but does not run tests probperly ("no tests found"). 

I've tried this in part by running tests and viewing the coverage table, but mostly through `ng test -- --showConfig` and checking the contents of `globalConfig`. 

**To Reproduce**
1. Set `coverage: false` and `collectCoverageFrom: [ 'src/app/**/*.service.ts' ]` in `angular.json`
2. Run `ng test -- --showConfig` and see that `globalConfig.coverage: false` and `globalConfig.collectCoverageFrom: null`
3. Set `coverage: true` and rerun the above command, see that `globalConfig.coverage` has flipped, and that _some_ of our settings apply
4. Run `ng test -- --collectCoverageFrom='[ 'src/app/**/*.service.ts' ]'` and see that output is `Unknwon option`
5. Run `jest --collectCoverageFrom='[ 'src/app/**/*.service.ts' ]' --showConfig` and see that `collectCoverageFrom` is set properly.

I'll see about providing a minimal repro ASAP, unless it's just me misunderstanding things. 

**Expected behavior**
I expect that setting `collectCoverageFrom` in `angular.json` will effectively set the same option in Jest, and thusly that it will limit the coverage report to the globbed files. 

**Screenshots**

**Builder:**
 - @angular-builders/jest:run
 - 7.2.0

**Libraries**
 - `angular-devkit/build-angular`: 0.11.4
-  `jest-preset-angular`: 6.0.2

**Additional context**

**Comment by emiljohnsen** [comment]
It seems my issues resolved themselves, or I resolved them, so this may be closed as operator error if you so wish. 

I was able to move config to `jest.config.js` (and `jest-ci.config.js` for an alternative config) and referring to the latter via CLI. By moving config to a config file, I was able to get `collectCoverageFrom` to work. 

I was still unable, however, to provide the options by CLI and via `angular.json`.
